pkg/cmd/server/api/v1/types.go

```diff
@@ -143,6 +143,14 @@ type MasterConfig struct {
 	// omitted) and controller election can be disabled with -1.
 	ControllerLeaseTTL int `json:"controllerLeaseTTL"`
 
+	// AdmissionPluginOverride is a list of admission control plugins that will be installed
+	// on the Kubernetes master. Order is significant. If empty, a default list of plugins is used.
+	AdmissionPluginOverride []string `json:"admissionPluginOverride,omitempty"`
```

Comment:
Got it.. I'm fine with that. I'll merge this with the other PR for config. And the entire AdmissionConfig struct should be omitempty?
Also, is it ok to have PluginOrderOverride instead of just PluginOrder so we know we can leave it blank?